app/javascript/packs/document-capture.tsx

```diff
@@ -97,6 +97,8 @@ const App = composeComponents(
     {
       value: {
         inPersonURL,
+        locationsURL: new URL('/verify/in_person/usps_locations', window.location.href).toString(),
+        addressSearchURL: new URL('/api/addresses', window.location.href).toString(),
```

Comment:
The pattern for other URLs is for the backend to pass them to the javascript app. I didn't do that here because it's a larger refactor, but I'm leaning towards creating a ticket for it

Reply:
eg [here](https://github.com/18F/identity-idp/blob/3fb9a2c099cd495b87d6e13299447b59006398bf/app/views/idv/shared/_document_capture.html.erb#L16) and [here](https://github.com/18F/identity-idp/blob/3fb9a2c099cd495b87d6e13299447b59006398bf/app/views/idv/shared/_document_capture.html.erb#L30)